OMFAPI/Endpoint.cs

```diff
@@ -17,12 +22,12 @@ public class Endpoint
         /// <summary>
         /// The name of the Namespace in OCS that is being sent to
```

Comment:
We might want to also update the comment to use 'Id' instead of 'Name'

Reply:
Changed